eventstream.go

```diff
@@ -416,7 +416,12 @@ func NewAuditLogBuilder() *AuditLogBuilder {
 
 type AuditLogPayload struct {
 	Content map[string]interface{} `json:"content"`
-	Diff    map[string]interface{} `json:"diff"`
+	Diff    AuditLogDiff           `json:"diff"`
+}
+
+type AuditLogDiff struct {
+	Before map[string]interface{} `json:"before,omitempty"`
+	After  map[string]interface{} `json:"after,omitempty"`
```

Comment:
if diff exists, then before & after should be required

Reply:
updated